vumi_http_proxy/test/test_http_proxy.py

```diff
@@ -0,0 +1,96 @@
+from twisted.internet import reactor
+from twisted.internet.defer import inlineCallbacks, Deferred, returnValue
+from twisted.internet.endpoints import serverFromString, clientFromString
+from twisted.trial import unittest
+from twisted.test import proto_helpers
+from twisted.web.client import ProxyAgent, readBody
+
+from vumi_http_proxy.http_proxy import ProxyFactory
+
+from .helpers import HttpTestServer, DEFAULT_TIMEOUT
+
+timeout = DEFAULT_TIMEOUT
+
+
+class ProxyToLocalServerTestCase(unittest.TestCase):
+
+    timeout
```

Comment:
You probably want `timeout = DEFAULT_TIMEOUT` here instead of `timeout`, and you can remove the module-level `timeout = DEFAULT_TIMEOUT`.